crates/polars-arrow/src/array/binview/view.rs

```diff
@@ -136,6 +136,28 @@ impl View {
         }
     }
 
+    /// Checks if the string starts with the prefix
```

Comment:
We don't want this compute in polars-arrow, I think this should be a function in `polars-ops`.